src/Symfony/Component/Validator/Constraints/ImageValidator.php

```diff
@@ -234,4 +238,29 @@ public function validate(mixed $value, Constraint $constraint)
             imagedestroy($resource);
         }
     }
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isSvg($value)
+    {
+        return 'image/svg+xml' === mime_content_type($value);
+    }
+
+    /**
+     * @param mixed $value
+     *
+     * @return array
+     */
+    private function getSvgSize($value)
+    {
+        $xmlValue = simplexml_load_file($value);
```

Comment:
what happens if the xml is corrupted?

Reply:
I added an extra check, if the xml is corrupted, we add CORRUPTED_IMAGE_ERROR